tests/integration/update.bats

```diff
@@ -69,7 +69,7 @@ EOF
         MEM_SWAP="memory.memsw.limit_in_bytes"
         SD_MEM_SWAP="unsupported"
         SYSTEM_MEM=$(cat "${CGROUP_MEMORY_BASE_PATH}/${MEM_LIMIT}")
-        SYSTEM_MEM_SWAP=$(cat "${CGROUP_MEMORY_BASE_PATH}/$MEM_SWAP")
+        SYSTEM_MEM_SWAP=$(FILE="${CGROUP_MEMORY_BASE_PATH}/${MEM_SWAP}"; test -f "$FILE" && cat "$FILE")
```

Comment:
why not 
```bash
SYSTEM_MEM_SWAP=""
if [[ -f ... ]]; then
  SYSTEM_MEM_SWAP=...
fi
```

Reply:
Not sure why is this any better than any of my two previous versions. To me, it actually looks worse for two reasons:
1 .file name has to be constructed twice.
2. four lines of code instead of one.

Or do you find `test -f "$FILE" && cat "$FILE"` not very readable?

Reply:
Nevermind, I'll fix it in a different way

Reply:
updated, PTAL @AkihiroSuda